karma.conf.js

```diff
@@ -17,7 +17,7 @@ module.exports = function(config) {
         'jquery.flot.flatdata.js',
         'jquery.flot.drawSeries.js',
         'jquery.flot.navigate.js',
-        'jquery.flot.time.js',
+        'jquery.flot.absRelTime.js',
```

Comment:
This file looks brand new. What happened to jquery.flot.time.js? Was it renamed? Did you plan to delete it?

Because the history of the first file is lost, the diff tool is showing that you added the entire absRelTime file.